src/css/footer.css

```diff
@@ -0,0 +1,12 @@
+footer {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  padding: 2em;
+  border: 2px solid black;
+}
```

Comment:
[OPTIONAL]
* the footer should move a bit down. I think it should be even closer to the bottom of the page. Try using the css position property to achieve this.